### PR TITLE
Updated folder and file structure as per instruction

### PR DESCRIPTION
**Pull request by emtec-damodhar-jadhao**

### CODE REVIEW

File: 6_September/Javascript_Assignment/script/Question1.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ //1) Create a function that takes two numbers as arguments and returns their sum.
+function addition(num1,num2){
+    return num1+num2
+}
+// const test1=addition(3, 2)
+// document.getElementById('main').innerHTML=test1
+// const test2=addition(-3, -6)
+// document.getElementById('main').innerHTML=test2
+const test3=addition(7, 3)
+document.getElementById('main').innerHTML=test3
```
Comment (nitish-poojari) [inline 6_September/Javascript_Assignment/script/Question1.js]: this should be in index.html

File: 6_September/Javascript_Assignment/script/Question10.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+            function odd_even(A){
+                if (A%2==0){
+                    return "Given Number is Even"
+                }else{
+                    return "Given Number is Odd"          
+                      }
+            }
+            // const number_check=odd_even(11)
+            const number_check= odd_even(100)
+            document.getElementById('main').innerHTML=number_check
```
Comment (nitish-poojari) [inline 6_September/Javascript_Assignment/script/Question10.js]: same for all

File: 6_September/Restaurant_Website/css/index.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.carousel-item img {
+    width: 100%;
+    height: 500px;
+}
```
Comment (nitish-poojari) [inline 6_September/Restaurant_Website/css/index.css:4]: keep single space between each css

File: 6_September/Javascript_Assignment/script/Question1.js (hunk 1 above)
Comment (emtec-damodhar-jadhao) [reply 6_September/Javascript_Assignment/script/Question1.js]: Updated file

File: 6_September/Restaurant_Website/css/index.css (hunk 3 above)
Comment (emtec-damodhar-jadhao) [reply 6_September/Restaurant_Website/css/index.css:4]: CSS Files updated

File: 6_September/Javascript_Assignment/script/Question10.js (hunk 2 above)
Comment (emtec-damodhar-jadhao) [reply 6_September/Javascript_Assignment/script/Question10.js]: updated changes

File: 6_September/Javascript_Assignment/script/Question10.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+            function odd_even(A){
+                if (A%2==0){
+                    return "Given Number is Even"
+                }else{
+                    return "Given Number is Odd"          
+                      }
+            }
+            // const number_check=odd_even(11)
+            const number_check= odd_even(100)
+            document.getElementById('ten').innerHTML=number_check
```
Comment (nitish-poojari) [inline 6_September/Javascript_Assignment/script/Question10.js]: you should not call the function from here. It should be called from html file.

File: 6_September/Javascript_Assignment/script/Question11.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    sum=0
+    for(i=0;i<=num;i++){
+        if(i%3==0 || i%5==0){
+            sum=sum+i
+        }       
+    } 
+    return sum  
+}
+const all_sum= multiples_of(10)
+document.getElementById('eleven').innerHTML=all_sum
```
Comment (nitish-poojari) [inline 6_September/Javascript_Assignment/script/Question11.js]: same everywhere

File: 6_September/Javascript_Assignment/script/Question10.js (hunk 4 above)
Comment (emtec-damodhar-jadhao) [reply 6_September/Javascript_Assignment/script/Question10.js]: Function call From Html File Using Event

File: 6_September/Javascript_Assignment/script/Question11.js (hunk 5 above)
Comment (emtec-damodhar-jadhao) [reply 6_September/Javascript_Assignment/script/Question11.js]: changes Updated

**Review by nitish-poojari — APPROVED** [review]

**Pull request merged**